pkg/ddevapp/ddevapp.go

```diff
@@ -1758,10 +1762,6 @@ Fix with 'ddev config global --required-docker-compose-version="" --use-docker-c
 		return err
 	}
 
-	if _, err = app.CreateSettingsFile(); err != nil {
-		return fmt.Errorf("failed to write settings file %s: %v", app.SiteDdevSettingsFile, err)
-	}
-
```

Comment:
I think we should have `app.CreateSettingsFile()` in both places, on pre-start and post-start.

I suggest to bring back this removed condition.

This code runs only when site is running:

https://github.com/ddev/ddev/blob/7b0cb1482c2596be0cbf67f3310883e4973992d7/pkg/ddevapp/drupal.go#L565-L582

Reply:
IMO it's pretty safe to do that, but here it was inside mutagenEnabled right?

Reply:
Yes, inside mutagenEnabled.

Reply:
I added it back where it was before (but that was late in Start, not in mutagen section)